starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -39,4 +39,7 @@ async def get_nonce(contract_address: Address) -> Felt:
     """
     Get the latest nonce associated with the given address
     """
-    raise NotImplementedError()
+    result = await state.starknet_wrapper.get_nonce(
```

Comment:
also covered in [PR](https://github.com/Shard-Labs/starknet-devnet/pull/292/files#diff-739e82de501883ee5e0f7646c74fcf1e0282e936dcf6e0eeb2a6700c5f74a307)